src/dotnet/APIView/APIViewWeb/Hubs/SignalRHub.cs

```diff
@@ -0,0 +1,36 @@
+using System.Threading.Tasks;
+using APIViewWeb.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Logging;
+
+namespace APIViewWeb.Hubs
+{
+    [Authorize]
+    public class SignalRHub : Hub
+    {
+        private readonly ILogger<SignalRHub> _logger;
+        public SignalRHub(ILogger<SignalRHub> logger) {
+            _logger = logger;
+        }
+
+        public override Task OnConnectedAsync()
+        {
+            string name = Context.User.Identity.Name;
+            if (!string.IsNullOrEmpty(name))
+            {
+                Groups.AddToGroupAsync(Context.ConnectionId, name);
+                Clients.Caller.SendAsync("ReceiveConnectionId", Context.ConnectionId);
+            }
+            return base.OnConnectedAsync();
+        }
+
+        public async Task ReceiveComment(string reviewId, string elementId, string partialViewResult)
```

Comment:
Might be nice to name the Hub Method with a different name than the client maybe CommentPush or Push Comments

Reply:
Good point, I changed the server side calls to `PushComment` and kept client side calls to `ReceiveComment` including the call made in `comments.ts`. Did you want it only for the Hub method?